script.js

```diff
@@ -116,3 +139,70 @@ function sendMessage(groupId, message){
         error: function () { }
     });
 }
+
+
+function onConnectFailed(resp) {
+    console.log("onConnectFailed triggered");
+    if (navigator.onLine) {
+        console.log("reconnecting");
+        window.Applozic.ALSocket.reconnect();
+    }
+}
+function subscribeToTypingChannel(groupId){
+    window.Applozic.ALSocket.subscibeToTypingChannel(groupId);
+}
+
```

Comment:
is this same? `subscribeToTypingChannelByGroupId `

Reply:
removed